tests/backup/backup_test.go

```diff
@@ -1129,6 +1129,214 @@ var _ = Describe("{ShareBackupAndEdit}", func() {
 	})
 })
 
+func createUsers(numberOfUsers int) []string {
```

Comment:
Move this function to the bottom of the file

Reply:
Moved to bottom of the file